doc/api/os.md

```diff
@@ -509,6 +509,29 @@ On POSIX systems, the operating system release is determined by calling
 available, `GetVersionExW()` will be used. See
 <https://en.wikipedia.org/wiki/Uname#Examples> for more information.
 
+## `os.guessHandleType(handle)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `handle` {integer} The handle number to try and guess the type of.
```

Comment:
This should really specify that "handle" means "file descriptor". In fact, given that the fs docs almost exclusively use "file descriptor" and "fd", I would also adopt that terminology here.

Reply:
Then I could also rename the function to `guessFileDescriptorType`? Or should I leave it as is?

And will apply the changes to fd/file descriptor in a few

Reply:
Yeah, I'd probably do that. In the context of libuv, `guessHandleType` makes more sense, because it returns a value that actually corresponds to the types of libuv handles, which wrap around fds.

Reply:
Done! Let me know if I've also done the rebase correctly 🤞